.github/workflows/linux_conformance.yml

```diff
@@ -0,0 +1,952 @@
+
+      - name: Build Conformance Targets
+        working-directory: ${{ env.BUILD_DIR }}
+        run:
+          make --jobs=$(nproc --all) subgraphsDumper
+          make --jobs=$(nproc --all) conformanceTests
+          make --jobs=$(nproc --all) apiConformanceTests
+
+      - name: Build
+        run: cmake --build ${{ env.BUILD_DIR }} --parallel ${{ steps.cpu-cores.outputs.count }} --config Release
+
+      - name: Show ccache stats
+        run: ccache --show-stats
+
+      - name: Cmake Layer Tests
+        run: cmake -GNinja -S ${{ env.OPENVINO_REPO }}/tests/layer_tests -B ${{ env.BUILD_DIR }}/layer_tests
+
+      - name: Cmake Model Hub Tests
+        run: cmake -GNinja -S ${{ env.OPENVINO_REPO }}/tests/model_hub_tests -B ${{ env.BUILD_DIR }}/model_hub_tests
+
+      - name: Build Layer Tests
+        run: cmake --build ${{ env.BUILD_DIR }}/layer_tests --parallel --config Release
+
+      - name: Build Model Hub Tests
+        run: cmake --build ${{ env.BUILD_DIR }}/model_hub_tests --parallel --config Release
+
+      - name: Install wheel packages
+        run: cmake -DCOMPONENT=python_wheels -DCMAKE_INSTALL_PREFIX=${{ env.INSTALL_DIR }} -P ${{ env.BUILD_DIR }}/cmake_install.cmake
+
+      - name: Install Layer Tests
+        run: cmake -DCOMPONENT=tests -DCMAKE_INSTALL_PREFIX=${{ env.INSTALL_DIR }} -P ${{ env.BUILD_DIR }}/layer_tests/cmake_install.cmake
+
+      - name: Install Model Hub Tests
+        run: cmake -DCOMPONENT=tests -DCMAKE_INSTALL_PREFIX=${{ env.INSTALL_DIR }} -P ${{ env.BUILD_DIR }}/model_hub_tests/cmake_install.cmake
+
+      - name: Install python wheels
+        run: python3 -m pip install openvino-dev --find-links=${{ env.INSTALL_DIR }}/tools
+
+      - name: Install tests
+        run: cmake -DCMAKE_INSTALL_PREFIX=${{ env.INSTALL_DIR }} -DCOMPONENT=tests -P ${{ env.BUILD_DIR }}/cmake_install.cmake
+
+      - name: Install OpenVINO
+        run: cmake -DCMAKE_INSTALL_PREFIX=${{ env.INSTALL_DIR }} -P ${{ env.BUILD_DIR }}/cmake_install.cmake
+
+      - name: CMake Samples Tests
+        run: cmake -GNinja -S ${{ env.OPENVINO_REPO }}/tests/samples_tests -B ${{ env.BUILD_DIR }}/samples_tests
+
+      - name: Build Samples Tests
+        run: cmake --build ${{ env.BUILD_DIR }}/samples_tests --config Release
+
+      - name: Install Samples Tests
+        run: cmake -DCOMPONENT=tests -DCMAKE_INSTALL_PREFIX=${{ env.INSTALL_DIR }} -P ${{ env.BUILD_DIR }}/samples_tests/cmake_install.cmake
+
+      - name: Pack Artifacts
+        run: |
+          pushd ${{ env.INSTALL_DIR }}
+            tar -czvf ${{ env.BUILD_DIR }}/openvino_package.tar.gz --exclude=tests *
+          popd
+
+          pushd ${{ env.INSTALL_DIR }}
+            tar -czvf ${{ env.BUILD_DIR }}/openvino_tests.tar.gz tests/
+          popd
+
+      - name: Check Dir 1
+        run: ls -laR ${{ env.BUILD_DIR }}
+
+      - name: Check Dir 2
+        run: ls -laR ${{ env.INSTALL_DIR }}
+
+      - name: Check Dir 3
+        run: ls -laR
```

Comment:
I do not see any runs.. Am I correct?

Reply:
It is still in the very early stages, I will re-request review once it is in a more ready state.

Reply:
Please, contact me to understand details